native/src/seal/encryptor.cpp

```diff
@@ -88,6 +88,21 @@ namespace seal
     void Encryptor::encrypt_zero_internal(
         parms_id_type parms_id, bool is_asymmetric, bool save_seed, Ciphertext &destination,
         MemoryPoolHandle pool) const
+    {
+        // Do not export noise.
+        StaticPolynomialArray u_destination;
+        StaticPolynomialArray e_destination;
+
+        encrypt_zero_internal(parms_id, is_asymmetric, save_seed, false, false, destination, u_destination, e_destination, pool);
```

Comment:
Shouldn't we disable the special modulus?

Reply:
I'll double check the logic when I am back at my computer but essentially I wanted to leave the code with equivalent functionality but with the option to turn off the special modulus. So the default methods (the ones that are called from other functions using the same syntax as before) do not change the special modulus functionality, while the methods with more arguments allow you to do so.

Reply:
I double checked the logic and ensured that the rust component that runs this code also gets a disabled modulus when calling the extended `encrypt` function.